kontrakter/src/main/kotlin/no/nav/pleiepengerbarn/uttak/kontrakter/Uttaksgrunnlag.kt

```diff
@@ -24,7 +24,8 @@ data class Uttaksgrunnlag (
     @JsonProperty("inngangsvilkår") val inngangsvilkår: Map<String, List<Vilkårsperiode>> = mapOf(),
     @JsonProperty("tilsynsperioder") val tilsynsperioder: Map<LukketPeriode, Duration> = mapOf(),
     @JsonProperty("beredskapsperioder") val beredskapsperioder: Map<LukketPeriode, Utfall> = mapOf(),
-    @JsonProperty("nattevåksperioder") val nattevåksperioder: Map<LukketPeriode, Utfall> = mapOf()
+    @JsonProperty("nattevåksperioder") val nattevåksperioder: Map<LukketPeriode, Utfall> = mapOf(),
+    @JsonProperty("kravprioritet") val kravprioritet: Map<LukketPeriode, Saksnummer> = mapOf()
```

Comment:
Map-et må gi en liste av saksnummere -- siden det er ikke sikkert at hele pleietiden blir brukt opp av én søker (og vi støtter flere enn to søkere). Endre til: 
`Map<LukketPeriode, List<Saksnummer>>`

Reply:
Ja, selvsagt. Gikk litt fort.

Reply:
Fikset.